cluster/images/conformance/OWNERS

```diff
@@ -5,10 +5,18 @@ reviewers:
   - dims
   - ixdy
   - spiffxp
+  - neolit123
+  - bentheelder
+  - brahmaroutu
+  - stevesloka
+  - johnSchnake
```

Comment:
@johnSchnake is not a @kubernetes member. John, could you [apply](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) to be a member? This should give you powers to lgtm. :)

Reply:
https://github.com/kubernetes/org/issues/557